Backend/src/main/java/ktsnvt/tim1/services/EventService.java

```diff
@@ -83,4 +82,42 @@ public EventDTO editEvent(EventDTO event) throws EntityNotFoundException, Entity
 
         return new EventDTO(eventRepository.save(e));
     }
+
+    public EventDTO setEventLocationAndSeatGroups(LocationSeatGroupDTO seatGroupsDTO) throws EntityNotFoundException {
```

Comment:
Method should be split into submethods to promote reusability and make testing easier

Reply:
Location changing needs to be implemented, as well as checks for existing reservations before saving any of the changes.